### DISCUSSION TITLE
Form表单如何获取点击不更新字段，并获取Input的子段名？

### DISCUSSION THREAD

**Discussion by strongyc** [ant-design/ant-design#33568] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
用户在点击Input后，切入图像文字识别，如何把识别的内容填写入对应的Input内，有事件支持吗？

### What does the proposed API look like?
用户在点击Input后，切入图像文字识别，如何把识别的内容填写入对应的Input内，有事件支持吗？感谢大佬。

**Comment by PeachScript** [comment]
字段名设置给 Input 的 id 或者 name，`onFocus` 的时候走 DOM API 拿一下即可
